.github/workflows/release.yml

```diff
@@ -149,7 +149,7 @@ jobs:
         shell: bash
         # Currently building in release mode, but we could also have debug builds for testing
         run: |
-          [ -f ~/.cargo/env ] && source ~/.cargo/env ; cargo build --release
+          [ -f ~/.cargo/env ] && source ~/.cargo/env ; cargo build --release --features indexer
```

Comment:
maybe stupid question, but why not simply build with `--all-features`?

Reply:
I would say for releases it's probably better to go the additive way, `--all-features` may contain unnecessary debug/profiling stuff.

Reply:
okay, thanks